src/common/mfu_io.c

```diff
@@ -1183,12 +1216,14 @@ int daos_ftruncate(mfu_file_t* mfu_file, off_t length)
     daos_off_t offset = (daos_off_t) length;
     int rc = dfs_punch(mfu_file->dfs, mfu_file->obj, offset, DFS_MAX_FSIZE);
     if (rc) {
-        MFU_LOG(MFU_LOG_ERR, "dfs_punch failed (%d)",
+        MFU_LOG(MFU_LOG_ERR, "dfs_punch failed (%d %s)",
```

Comment:
Nice catch!

Reply:
Only because I was trying to copy/paste :)